### PR TITLE
[JENKINS-74017] Extract inline script block from `BapSshPublisherPlugin/config.jelly`

### PR DESCRIPTION
**Pull request by yaroslavafenkin**
https://issues.jenkins.io/browse/JENKINS-74017

### Testing done
Added 2 SSH servers on the Global Configuration page. Went to create a freestyle project. The affected script block is executed upon entering the page. I've checked whether the values and value types stored inside the global variable are the same before and after the change.

[Before the change](https://www.youtube.com/watch?v=pPnmeyc4Dl0)
[After the change](https://www.youtube.com/watch?v=sAnzTjISijc)

### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/webapp/js/pos.js [hunk 1]
```diff
@@ -106,4 +106,12 @@ var sshBehave = {
...
 
-Behaviour.register(sshBehave);
\ No newline at end of file
+Behaviour.register(sshBehave);
+
+window.addEventListener("DOMContentLoaded", function() {
+    document.querySelectorAll(".pos-host-configs-data-holder").forEach(function(dataHolder) {
+        const { configName, effectiveDisableExec } = dataHolder.dataset;
+
+        exec_disabled[configName] = effectiveDisableExec === "true";
```
Comment (yaroslavafenkin) [inline src/main/webapp/js/pos.js:115]: There doesn't seem to be more correct way to convert string to boolean. We don't expect any values other than `"true"` or `"false"`, so this should be good enough.
Could alternatively use `JSON.parse(effectiveDisableExec)` here. Let me know if you like it more and I should change it.

**Review by basil — APPROVED** [review]
Thanks!

**Comment by basil** [comment]
@gmcdonald Can this please be released? That will get us one step closer to marking this plugin as finished in the [CSP compatibility project tracking spreadsheet](https://docs.google.com/spreadsheets/d/1nSx4gQ2YUl-sKdY5u6biXmyDnkQ26wTaMBLSR8gr_-A/edit). If that is not practical, please let me know if I should adopt this plugin. Thanks!

**Comment by gmcdonald** [comment]
Thanks @basil, I will see about getting it done , will aim for the weekend if that is ok ?

**Comment by basil** [comment]
Sure, thank you!

**Comment by basil** [comment]
@gmcdonald Gentle ping that we're still interested in a release. BTW setting up https://www.jenkins.io/doc/developer/publishing/releasing-cd/ would make this less of a personal effort.

**Comment by gmcdonald** [comment]
Thanks @basil will try and migrate to CD releasing tomorrow. My only concern was about how to interact with the parent publish-over plugin if that affects things at all when it comes time to release a new version of that (and perhaps even a new release of that could auto-bump dependant plugins with its new version number !?)

**Comment by basil** [comment]
There shouldn't be any negative interactions with dependent plugins when adopting CD. Dependabot is enabled for this repository, so you should get a dependabot PR when the parent plugin is released again.

**Comment by gmcdonald** [comment]
PRs created for adopting CD - 

https://github.com/jenkinsci/publish-over-ssh-plugin/pull/363
https://github.com/jenkins-infra/repository-permissions-updater/pull/4181

**Pull request merged**